db/seeds.rb

```diff
@@ -1,7 +1,29 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'faker'
+
+
+
+50.times do
+  pw = Faker::Internet.password(8)
+  User.create!(
+
+  name: Faker::Name.name,
```

Comment:
may want to indent these lines since its part of the argument list for `User.create`